.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     - uses: actions/setup-python@v5
       with:
-        python-version: ">= 3.7"
+        python-version: ">= 3.8"
```

Comment:
NB: Ideally this and others would be `python-version-file` instead, but that requires #156.

(I'll take a look at reviving that as well.)